src/OrderFormAcceptanceTests.Steps/Utils/EnvironmentVariables.cs

```diff
@@ -54,6 +54,15 @@ internal static string DbConnectionString()
             return string.Format(ConnectionString.GPitFutures, serverUrl, databaseName, dbUser, dbPassword);
         }
 
+        internal static string BapiDbConnectionString()
+        {
+            var (serverUrl, databaseName, dbUser, dbPassword) = DbConnectionDetails();
+
+            databaseName = Environment.GetEnvironmentVariable("BAPIDATABASENAME") ?? "buyingcatalogue";
+
+            return string.Format(ConnectionString.GPitFutures, serverUrl, databaseName, dbUser, dbPassword);
```

Comment:
How are we differentiating the server url for ORDAPI DB and BAPI DB? it looks like they come from the same place

Reply:
we're not, server url is the same. It's just the database name

Reply:
But it may not be, the port changing could cause an error (i.e. 1450 for bapi, 1452 for ordapi)

Reply:
Adrian's change to have all the DBs in one sql instance means they're all running on port 1450 now. Greatly simplifies this

Reply:
ah, wasn't fully aware of that